lib/membrane/video_compositor/multiple_inputs/video_compositor/implementation.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.VideoCompositor.MultipleInputs.VideoCompositor.Implementation do
```

Comment:
- `VideoCompositor.MultipleInputs.VideoCompositor` 🤔 I'd generally try to shorten module names
- Isn't it supposed to replace the current compositor?
- I think this should be in a separate PR

Reply:
- See the following subsection. I will change in the future. I will leave it as it is, right now.
 - We decided to separate these two approaches. In future PRs, we may remove one of them or merge them.
 -  I use it in the following PR, implementing Rust OpenGL, I think it should stay.